server.go

```diff
@@ -115,9 +115,11 @@ func (a *App) handleFromLocalItem(ctx context.Context, msg Message, dst int) err
 
 func (a *App) handleFromLocal(ctx context.Context, msg Message) error {
 	for _, dst := range msg.TwinDst {
-		if err := a.handleFromLocalItem(ctx, msg, dst); err != nil {
-			log.Error().Err(err).Msg("failed to handle message in handle_from_local_prepare")
-		}
+		go func(dst int) {
```

Comment:
Why this is needed, this is already running inside the worker context. this means u are spawnning more go routines (unmanaged) than the managed workers.